lib/features/recommendation/widgets/organisation_header.dart

```diff
@@ -32,40 +34,42 @@ class OrganisationHeader extends StatelessWidget {
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.start,
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: organisation.tags
-                    .map(
-                      (tag) => Container(
-                        margin: const EdgeInsets.symmetric(vertical: 3),
-                        decoration: BoxDecoration(
-                          color: tag.color,
-                          borderRadius: const BorderRadius.only(
-                            topRight: Radius.circular(25),
-                            bottomRight: Radius.circular(25),
-                          ),
-                        ),
-                        child: Padding(
-                          padding: const EdgeInsets.symmetric(
-                            vertical: 5,
-                            horizontal: 15,
+              child: SingleChildScrollView(
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: organisation.tags
+                      .map(
+                        (tag) => Container(
+                          margin: const EdgeInsets.symmetric(vertical: 3),
+                          decoration: BoxDecoration(
+                            color: tag.color,
+                            borderRadius: const BorderRadius.only(
+                              topRight: Radius.circular(25),
+                              bottomRight: Radius.circular(25),
+                            ),
                           ),
-                          child: Text(
-                            tag.displayText,
-                            style: TextStyle(
-                              color: Colors.white,
-                              fontWeight: FontWeight.bold,
-                              fontSize:
-                                  MediaQuery.of(context).size.height * 0.025,
+                          child: Padding(
+                            padding: const EdgeInsets.symmetric(
+                              vertical: 5,
+                              horizontal: 15,
+                            ),
+                            child: Text(
+                              tag.displayText,
+                              style: TextStyle(
+                                color: Colors.white,
+                                fontWeight: FontWeight.bold,
+                                fontSize: FontUtils.getScaledFontSize(
+                                    inputFontSize: 15, size: size),
+                              ),
                             ),
                           ),
                         ),
-                      ),
-                    )
-                    .toList()
-                    .take(3)
-                    .toList(),
+                      )
+                      .toList()
```

Comment:
Thanks, fixed
I tried to do not touch a lot organisations and organisation details features (only tiny visible fixes) to safe a time since they was out of scope. It's just a general context)